bip32.c

```diff
@@ -94,7 +99,7 @@ int hdnode_from_xprv(uint32_t depth, uint32_t child_num, const uint8_t *chain_co
 	const curve_info *info = get_curve_by_name(curve);
 	if (info == 0) {
 		failed = true;
-	} else {
+	} else if (info->params) {
```

Comment:
It's only used with Ed25519 in the unit tests so it's not exploitable. But I will make a new pull request for it.